src/components/Heading/Heading.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+export type HeadingLevelType = "h2" | "h3" | "h4" | "h5";
+
+type HeadingProps = {
+  children: React.ReactNode;
+  level?: HeadingLevelType;
+  className?: string;
+};
+
+const Heading: React.FunctionComponent<HeadingProps> = ({
+  children,
+  level = "h2",
```

Comment:
Please remove the default value for this component.

Having a proper heading structure is important and using default values prevents the developer from having to take it into consideration.